docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.deploy.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @file
+ * Drush deploy hooks for hs_paragraph_types module.
+ */
+
+/**
+ * Copy old background color field data to new background field.
+ */
+function hs_paragraph_types_deploy_10000() {
```

Comment:
@codechefmarc SWS don't use deploy hooks (see discussion [here](https://github.com/SU-HSDO/suhumsci/pull/1425#discussion_r1459737628) for the backstory). Can you please switch to an update hook?  

Update hooks run before the config import, so it will be necessary to import the config for the new field programmatically, before updating the values. There are multiple hooks in `hs_paragraph_types.install` that do this, in case you need an example.

Also, can you please use batching (`$sandbox`) for processing the updates, to prevent any possible issues on deployment.

Reply:
Ok, I can rework this.

Reply:
Ok, fixed. Please re-test.